app/config/config.yml

```diff
@@ -248,3 +248,19 @@ services:
         public: false
         tags:
             - { name: twig.extension }
+
+    ForkCMS\Google\TagManager\DataLayer:
+        public: true
+
+    ForkCMS\Google\TagManager\TagManager:
+        public: true
+        arguments:
+            - "@fork.settings"
+            - "@ForkCMS\\Google\\TagManager\\DataLayer"
+            - "@ForkCMS\\Privacy\\ConsentDialog"
+
+    ForkCMS\Privacy\ConsentDialog:
+        public: true
+        arguments:
+            - "@fork.settings"
+            - "@fork.cookie"
```

Comment:
new line at the end of the file is missing

Reply:
Fixed in 1383ce812dd095e9da37e47f53d0623538c2f1c5